### ISSUE TITLE
GPS tracks are not displayed on the map, the problem with OSM is due to "Cyrillic"

### ISSUE THREAD

**Issue by torerobo** [openstreetmap/openstreetmap-website#2300]
If I load a gps-track (web browser), add it to Cyrillic (Russian), then the track after loading will not be displayed among the tracks on the OSM map layer. The problem is solved by changing the description from Cyrillic to Latin. But many know about it? (foreigners, please do not write that en-international language is just your imagination).

I checked the track was not displayed (description in Cyrillic) and after 3 weeks. As soon as I changed the description from Cyrillic to Latin, the track began to appear on the OSM map /
[link](https://help.openstreetmap.org/questions/69903/gps-tracks-are-not-displayed-on-the-map-the-problem-with-osm-is-due-to-cyrillic)
![1](https://user-images.githubusercontent.com/35532898/60793541-9170ab00-a170-11e9-8928-e6fb3939a069.jpg)

**Comment by tomhughes** [comment]
What do you mean by "displayed on the OSM map layer" exactly? Can you explain exactly what it is you're turning on where you're expecting the track to appear and it isn't?

**Comment by tomhughes** [comment]
Do you mean that you are selecting "Public GPS Traces" in the layer list here:

![image](https://user-images.githubusercontent.com/147741/60798104-6c783a00-a168-11e9-855b-ac3eec16fce4.png)

**Comment by torerobo** [comment]
> Вы имеете в виду, что вы выбираете «Public GPS Traces» в списке слоев здесь:
> 
> ![образ](https://user-images.githubusercontent.com/147741/60798104-6c783a00-a168-11e9-855b-ac3eec16fce4.png)

yes 
![1](https://user-images.githubusercontent.com/35532898/60802198-11067600-a181-11e9-86f7-09694cda9ab6.jpg)

**Comment by tomhughes** [comment]
Not sure why that would be upset by Cyrillic and I'm also quite surprised that it noticed the change you made because I wouldn't expect it to reconsider a trace.

The problem won't be in this code though, it will be in the service that renders those tiles, which is largely unmaintained these days.

**Comment by tomhughes** [comment]
Can you provide the ID of the track that caused the problem?

**Comment by torerobo** [comment]
However. Several tracks that were not displayed in public tracks were uploaded to the server (the description is in Cyrillic). After changing the description (in Latin), the tracks appeared in the public nka map.
Thank you, but I did everything I could.

And where to find the ID / can not see? (tracks hidden "private / anonymous")

Here are the tracks, an example that lagged because of the Cyrillic alphabet (now in Latin)
[Example ](https://www.openstreetmap.org/query?lat=55.31674&lon=36.73637#map=19/55.31679/36.73684&layers=G)No. 1 link to the map (incl. Public tracks) (single track).
[Example 2](https://www.openstreetmap.org/#map=19/55.76813/38.72956&layers=G) (single track)

**Comment by tomhughes** [comment]
Just give me the URL of the page for the track (should be something like https://www.openstreetmap.org/user/TomH/traces/3026889) and I can investigate what happened.

**Comment by tomhughes** [comment]
If you go to https://www.openstreetmap.org/traces and then select "See my traces" then you should get a list of the traces and you can click on the one with the problem and get the URL.

**Comment by torerobo** [comment]
https://www.openstreetmap.org/user/pangimun/traces/3040714
https://www.openstreetmap.org/user/pangimun/traces/3040727
I apply.
But now everything is in order with them, I changed the description to Latin.

**Comment by tomhughes** [comment]
Well those are both marked as private so I wouldn't expect either of them to show up on the "Public GPS Traces" layer... I suspect what you are seeing is not those traces at all but rather some other similar trace. Unless you have changed the visibility of the traces?

**Comment by torerobo** [comment]
> Ну, оба они помечены как частные, поэтому я не ожидаю, что кто-либо из них будет отображаться на слое «Public GPS Traces» ... Я подозреваю, что вы видите не эти следы вообще, а скорее какой-то другой аналогичный след. Разве вы не изменили видимость следов?

Yes, private. And they are displayed in public tracks. I have attached you links to tracks with an increase in a particular locality (only one track passes there and this track **is mine**)

**Comment by tomhughes** [comment]
Well they can't have been private when they were originally uploaded or they would never have been rendered there.

I can see the data for both on the tile server, and it appears have been fetched at the time they were originally uploaded so I would be very surprised if they weren't rendered at that point - are you sure you didn't just have cached tiles from looking at that area before the new traces had rendered?

**Comment by tomhughes** [comment]
Indeed I can see the second one being processed without any sign of an error:

```
Jul 06 07:52:59 noquiklos.openstreetmap.org update[1099]: [2019-07-06T07:52:59Z] INFO: GPX file tracks/pending/3040727.gpx
Jul 06 07:52:59 noquiklos.openstreetmap.org update[1099]: [2019-07-06T07:52:59Z] INFO: Attaching first point
Jul 06 07:52:59 noquiklos.openstreetmap.org update[1099]: [2019-07-06T07:52:59Z] INFO: GPX contained 5518 good point(s) and 0 bad point(s)
```

**Comment by torerobo** [comment]
> Well they can't have been private when they were originally uploaded or they would never have been rendered there.
> 
> I can see the data for both on the tile server, and it appears have been fetched at the time they were originally uploaded so I would be very surprised if they weren't rendered at that point - are you sure you didn't just have cached tiles from looking at that area before the new traces had rendered?

When I was looking for a problem. So I perezalival tracks in a new way. And maybe he made the tracks "public / Latin", and then again changed them to "anonymous / Latin"
There was no cache. I work in the browser in incognito mode

**Comment by torerobo** [comment]
Perhaps there is no problem with the tracks.
Initially, I downloaded all the tracks "anonymous / Cyrillic".
I didn’t find them on the map, I perezalivat tracks "public / Cyrillic" and again did not detect them (but here an important point: after processing, they are displayed on the map first on an enlarged scale). After I filled the track "public / Latin" and everything became normal.
In general, I will need to test the loading of tracks with Cyrillic and report the result.
Thank you, I have everything.
